client/http/types.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2023 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package http
+
+import "time"
+
+// NOTICE: the structures below are copied from the PD API definitions.
+// Please make sure the consistency if any change happens to the PD API.
+
+// RegionInfo stores the information of one region.
+type RegionInfo struct {
```

Comment:
Is it possible to share the same definition with api package?

Reply:
Because we cannot reference each other between the tikv/pd and tikv/pd/client packages, it probably won't work if we don't open a separate package to define these types.